### ISSUE TITLE
Use the default program for .ini files to edit Sandboxie.ini instead of always using Notepad

### ISSUE THREAD

**Issue by Dyras** [sandboxie-plus/Sandboxie#1475]
**Is your feature request related to a problem? Please describe.**
Hello! As it is now, Notepad is the only program used to open the Sandboxie.ini from the settings. I don't know if it's related, but this also has the annoying side effect of making it impossible for non-admin users to edit Sandboxie.ini, even if you enter the admin account's password into the UAC screen.

"Process is already running" is what it tells me. This seems to be due to the password protection, which doesn't seem to take into consideration that I'm a standard user using another account's admin privileges to edit Sandboxie.ini

**Describe the solution you'd like**
I wish Sandboxie either asked which program you want to use, or uses the default program for .ini files.

**Additional context**
tl;dr: I want the ability to use Notepad++ to edit Sandboxie.ini.

**Comment by Coverlin** [comment]
@Dyras try https://www.binaryfortress.com/NotepadReplacer/ and forget about windows notepad

**Comment by Dyras** [comment]
@Coverlin even with that installed, Sandboxie-Plus uses Notepad.

**Comment by DavidXanatos** [comment]
in the next build in the sandboxie-plus.ini will be the setting
[Options]
Editor=notepad.exe
allowing to change the default editor by providing a file located in the path or a full path to the editor binary

**Comment by bastik-1001** [comment]
To me, it sounds like a good solution to be able to configure the editor to be used instead of the system default being used.

**Comment by isaak654** [comment]
If you plan to add a full path, it needs to be escaped in this way:

> [Options]
`Editor=C:\\Program Files (x86)\\Notepad++\\notepad++.exe`
>
> first you need to escape the \ characters like in my example
second SandMan.exe caches the entries so if you don't set a value using the UI
you need to restart the process for manual changes to take effect. 
>
> Sandboxie-Plus.ini will not have Editor=anything untill you add it using an editor of some sorts, it does not create entries with default values.

Source: https://www.wilderssecurity.com/threads/sandboxie-plus-1-0-7.443250/#post-3060802
